_docs_integrate/requests-via-relationshiptemplates.md

```diff
@@ -27,107 +27,141 @@ required_by:
+To do this, the Request must be specified in the `content` property of the payload.
+As an example in this guide, a [Request]({% link _docs_integrate/data-model-overview.md %}#request) containing only a single [AuthenticationRequestItem]({% link _docs_integrate/data-model-overview.md %}#authenticationrequestitem) within its `items` property is used, but any other Request can also be used.
 
-```json
+```jsonc
 {
   "content": {
+    // Specification of a Request
+    "title": "Example of a Request",
     "items": [
       {
         "@type": "AuthenticationRequestItem",
         "mustBeAccepted": true,
-        "title": "The Sender is asking for an authentication."
+        "title": "The Sender is asking for an authentication"
       }
     ]
   }
 }
 ```
 
-Even though the Requests are validated during the RelationshipTemplate creation you should not skip this step as it gives you additional information in case of validation errors.
+If a Request is contained within a RelationshipTemplate, its validity is automatically checked when the [RelationshipTemplate is created](#create-the-relationshiptemplate). The RelationshipTemplate cannot be created if the Request is faulty.
+Nevertheless, the Request's validity should be checked before attempting to create the RelationshipTemplate in order to obtain additional information about the reasons for the error in the case of a faulty Request.
 {: .notice--info}
 
 ## Create the RelationshipTemplate
 
-Next, [create a RelationshipTemplate]({% link _docs_use-cases/use-case-transport-create-own-relationshiptemplate.md %}) on the Sender Connector, using a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent).
-If there is no Relationship between the Connectors, yet, you must put the `content` property of the payload of the previous step in the `onNewRelationship` property of the RelationshipTemplateContent.
-However, if the Connectors already have a Relationship with each other, you must put it in the `onExistingRelationship` property of the `RelationshipTemplateContent`.
-But note, that the `onNewRelationship` property is required and must, therefore, always be set.
+Next, the Sender wants to create the RelationshipTemplate, which contains the Request it wants to send to the Recipient. To specify a Request within a RelationshipTemplate, a data object of type [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) must be used within the `content` property of the [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate).
+A Request can be specified in the `onNewRelationship` property or the `onExistingRelationship` property of the RelationshipTemplateContent.
+If no Relationship has yet been established between the Sender and the Recipient, the Recipient receives the Request specified in the `onNewRelationship` property when loading the RelationshipTemplate.
```

Comment:
Are you sure about that? I have now changed the future tense to present tense because of another comment you made. Is there a particular reason why future tense would be more appropriate here than present tense?